SQRLDotNetClientUI/Models/INotifyIcon.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Avalonia.Controls;
+using System;
+
+namespace SQRLDotNetClientUI.Models
```

Comment:
Very nice I love the interface / abstraction idea!

Reply:
Cool, glad you like it! 😁